ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -215,7 +215,10 @@ import RevenueCat
     @objc(showStoreMessagesForTypes:completion:)
     static func showStoreMessages(forRawValues rawValues: Set<NSNumber>,
                                   completion: @escaping () -> Void) {
-        Self.sharedInstance.showStoreMessages(forRawValues: rawValues, completion: completion)
+        let storeMessageTypes = rawValues.map { number in
```

Comment:
I think this should be
```suggestion
        let storeMessageTypes = rawValues.compactMap { number in
```